README.md

```diff
@@ -3,6 +3,7 @@ A collection of interesting projects and connected people that work on them
 
 Project | Short Description | Technlogies | People
 ------------ | ------------- | ------------- | -------------
+[30 Seconds of C++](11) | A collection of C++ STL features (functions/libraries) which can be learned in 30 seconds or less | C++/Documentation | [@thamara](8)
```

Comment:
```suggestion
[30 Seconds of C++](11) | A collection of C++ STL features (functions/libraries) which can be learned in 30 seconds or less | C++ - Documentation | [@thamara](8)
```

Not sure yet if I prefer `/` or `-`, but since all others are using `-`, let's keep the consistency hahaha